camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/EtlCounts.java

```diff
@@ -229,6 +235,10 @@ public void postTrackingCountToKafka(Configuration conf, String tier, String bro
     log.info(topic + " sent " + counts + " counts");
   }
 
+  protected String getMonitoringEventClass(Configuration conf) {
+    return conf.get(MONITORING_EVENT_CLASS);
+  }
```

Comment:
put `conf.get(MONITORING_EVENT_CLASS)` as a method, so that it can be overridden in subclass EtlCountsForUnitTest.